packages/flutter_tools/test/general.shard/fuchsia/fuchsia_device_test.dart

```diff
@@ -534,7 +534,7 @@ void main() {
 
       try {
         await device.takeScreenshot(globals.fs.file('file.ppm'));
-      } catch (_) {
+      } on Exception catch (_) {
```

Comment:
nit don't need to catch unused exception

Reply:
Done